Domain/Scheduling/CommandScheduler{T}.cs

```diff
@@ -14,18 +14,18 @@ internal class CommandScheduler<TAggregate> :
         ICommandScheduler<TAggregate>
         where TAggregate : class, IEventSourced
     {
-        protected readonly IEventSourcedRepository<TAggregate> repository;
+        protected readonly ICommandApplier<TAggregate> Target;

```

Comment:
Lower-case for fields. Probably no need for this to be protected. And the name should be, e.g., `commandApplier`.